### ISSUE TITLE
Bug: 

### ISSUE THREAD

**Issue by dfsbb** [sbb-design-systems/sbb-angular#459]
**Describe the bug**
When I click a button (any mode), after the click the state of the button should go back to default. At the moment after the click the button state stays on hover/focus. Only when I take the focus off the button the state changes. That's in my opinion not correct.

Affected are: public library, business library

**To Reproduce**
1. https://angular.app.sbb.ch/business/components/button
2. click on the button
3. after the click the state of the button is still hover/focus

see also:  ![button_after_click_not_correct_state](https://user-images.githubusercontent.com/45092436/84747544-b217c680-afb7-11ea-9594-bcc6da3ec686.gif)

**Expected behavior**
After the click the state of the button should go back to default state.

**Comment by kyubisation** [comment]
In my opinion the focused state (which this is), should also be highlighted.
@Frenggi What is your opinion on this?

**Comment by dfsbb** [comment]
@kyubisation yes you're right. Maybe I explained it not exactly. ;-) What I mean is, that after a click on a button the focus should not stay on this button. See sbb.ch behavior of the timetable search.

**Comment by kyubisation** [comment]
You could easily do this by setting the focus in the (click) event handler on the desired element, which should have the desired effect.
Or am I still misunderstanding something?
Because from an accessibility aspect it's not the best design to simply remove the focus.

**Comment by dfsbb** [comment]
Okay, I see...
So maybe we should think about to change the focus state styling?
https://digital.sbb.ch/de/webapps/components/button onClick looks like the hover state. But should the focus state really look exactly the same as the onClick and hover state? I have never seen that in forms before, that a clicked button still looks like clicked or hovered after the click. ;-)
@Frenggi, what do you think?

**Comment by Frenggi** [comment]
As already discussed bilateral with @dfsbb, I agree that the button should not look the same after a click.
When I check other design systems (e.g. [material design](https://material.io/components/buttons#contained-button)) the "focused" and the "onclick" / "pressed" are separated visually. I think we should do this as well. And then the button stays focused after click.

**Comment by mhaertwig** [comment]
@mcilurzo there's no design specification for this. Can you have a look, please?
